t/05-scheduler-full.t

```diff
@@ -486,6 +488,23 @@ EOC
     ok(Mojolicious::Commands->start_app('OpenQA::WebAPI', 'eval', '1+0'));
 }
 
+sub setup_share_dir {
```

Comment:
Makes sense to have it in https://github.com/os-autoinst/openQA/blob/master/t/lib/OpenQA/Test/Utils.pm so we can reuse it ! 👍